pygmt/src/blockm.py

```diff
@@ -111,6 +112,13 @@ def blockmean(data=None, x=None, y=None, z=None, outfile=None, **kwargs):
 
     {I}
 
+    summarize: str
+        [**m**\|\ **n**\|\ **s**\|\ **w**].
+        Type of summary values calculated by blockmean. Use **n** to report the
+        number of input points inside each block, **s** to report the sum of
+        all z-values inside a block, **w** to report the sum of weights
+        [Default or **m** reports mean value].
```

Comment:
List is more readable.
```suggestion
        Type of summary values calculated by blockmean. 
        
        - **n**: report the number of input points inside each block
        - **s**: report the sum of all z-values inside a block
        - **w**: report the sum of weights
        - **m**: report mean value [Default]
```